### ISSUE TITLE
Quest: The Summoning

### ISSUE THREAD

**Issue by underlord-oberon** [plantain-gnocchis/wow-voiceover-tbc-wotlk#55]
Add-on version: AI_VoiceOver_v1.4.5
Data version: AI_VoiceOverData_VanillaExtra_v1.4

Alterac Mountains / Chillwind Point / NPC: Bath'rah the Windwatcher / Quest: The Summoning
Link: https://www.wowhead.com/wotlk/quest=1713/the-summoning

Description: the voice-over suddenly silences during playback.

Progress: no playback.

Completion: playback ok.

**Comment by carevass** [comment]
This should be fixed in AI_VoiceOverData_VanillaExtra_v1.5.
